widgetastic_manageiq/__init__.py

```diff
@@ -1022,6 +1022,49 @@ def workaround_save_issue(self):
         )
 
 
+class ReactCodemirror(Widget):
+    """Looks the same as ScriptBox but it changede in 5.11
+    https://github.com/JedWatson/react-codemirror"""
+
+    def __init__(self, parent, locator=None, read_locator=None, logger=None):
```

Comment:
Q: It is not possible to take as single `locator` which point div of `CodeMirror` widget? and with help of this we can create other required locators like `read_locator`?

Reply:
hey, I think I will remove read_locator as it's more complicated as I originally thought.

However, there's no way you can build it from the main locator

```
locator="//div[contains(@class,'miq-codemirror')]//textarea",
read_locator="//pre[@class=' CodeMirror-line ']/span"
```
as you see, they are not related,and you send_keys in one place but read from another